src/test/ui/const-generics/defaults/const-default.rs

```diff
@@ -0,0 +1,14 @@
+// check-pass
+
+#![feature(const_generics)]
+#![allow(incomplete_features)]
+
+
+#[derive(Default)]
+pub struct ConstDefault<const N: usize = 3usize> {
```

Comment:
Does it work if you use `3` instead of`3usize`?

Reply:
ah the issue is that it ICEs before it reaches this point so, neither works currently